src/overlays/actors/ovl_En_Toto/z_en_toto.c

```diff
@@ -184,15 +184,15 @@ void EnToto_Init(Actor* thisx, PlayState* play) {
 
     Actor_ProcessInitChain(&this->actor, sInitChain);
     Collider_InitAndSetCylinder(play, &this->collider, &this->actor, &sCylinderInit);
-    if (play->sceneNum == SCENE_MILK_BAR &&
+    if (play->sceneId == SCENE_MILK_BAR &&
```

Comment:
```suggestion
    if ((play->sceneId == SCENE_MILK_BAR) &&
```
And other sceneId checks in this file